jax/experimental/jax2tf/tests/correctness_stats.py

```diff
@@ -69,16 +70,59 @@ def _to_np_dtype(dtype):
       tf_unimpl(f"{prim.name} is unimplemented for dtype {np_dtype}")
```

Comment:
It seems that it may be useful to change `tf_unimpl` to take just the `np_dtype` as an argument and keep the rest constant.

Reply:
Addressed in `c289a06`